crates/host/src/wasmbus/mod.rs

```diff
@@ -3374,14 +3071,13 @@ impl Host {
         Ok(res.into())
     }
 
-    // #[instrument(level = "debug", skip_all)] // FIXME: this is temporarily disabled because wadm (as of v0.8.0) queries links too often
+    #[instrument(level = "debug", skip_all)]
     async fn handle_links(&self) -> anyhow::Result<Bytes> {
-        trace!("handling links"); // FIXME: set back to debug when instrumentation is re-enabled
+        debug!("handling links");
 
         let links = self.links.read().await;
-        let links: Vec<LinkDefinition> = links.values().cloned().collect();
-        let res = serde_json::to_vec(&LinkDefinitionList { links })
-            .context("failed to serialize response")?;
+        let links: Vec<InterfaceLinkDefinition> = links.values().flatten().cloned().collect();
```

Comment:
Given that this is a fairly common call for wadm, I think we should avoid the clone here and just get a vec of borrowed linkdefs. Serde only needs to borrow the data here anyway